applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_adjoint_potential_flow_element.cpp

```diff
@@ -411,5 +412,6 @@ namespace Kratos
     // Template class instantiation
 
     template class AdjointIncompressiblePotentialFlowElement<IncompressiblePotentialFlowElement<2,3>>;
+    template class AdjointIncompressiblePotentialFlowElement<CompressiblePotentialFlowElement<2,3>>;
```

Comment:
This is really confusing. If it is intentional, please document it with a comment.

Reply:
This was solved by @marcnunezc here https://github.com/KratosMultiphysics/Kratos/pull/4662 and by me here https://github.com/KratosMultiphysics/Kratos/pull/4642/commits/accaa4b821a0e004a29c875861df4fb09e5d33c4